webservices/tasks/legal_docs.py

```diff
@@ -30,6 +43,35 @@ def refresh():
         refresh_aos(conn)
         refresh_murs(conn)
 
+@app.task(once={'graceful': True}, base=QueueOnce)
+def reload_all_aos():
+    """
+    Reload all AOs daily (determine by schedule) if there were any new or modified AOs found for the past 24 hour period
+    """
+    with db.engine.connect() as conn:
+        row = conn.execute(DAILY_MODIFIED_STARTING_AO).first()
+        if row:
+            logger.info("AO found %s modified at %s", row["ao_no"], row["pg_date"])
+            logger.info("Daily (%s) reload of all AOs starting", datetime.date.today().strftime("%A"))
+            load_advisory_opinions()
+            logger.info("Daily (%s) reload of all AOs completed", datetime.date.today().strftime("%A"))
+            slack_message = 'Daily reload of all AOs completed in {0} space'.format(env.get_credential('NEW_RELIC_APP_NAME'))
```

Comment:
It was confusing to me that we are using 'NEW_RELIC_APP_NAME'. It may be better to have a utility method like `get_app_name()`. Inside that method, we could add a comment saying that we leverage 'NEW_RELIC_APP_NAME'.